readme.md

```diff
@@ -31,44 +30,34 @@ But, you may be asking yourselves, "Why do we care?" <pause>
 
 You tell me.  Check out these demos.
 
-- [isotope](http://codepen.io/desandro/full/nFrte)
 - [tablesorter](http://tablesorter.com/docs/#Demo)
+- [isotope](http://codepen.io/desandro/full/nFrte)
 
 Q. Why do we care?
 ---
 
 > A. Encapsulation of really cool functionality, so we can reuse and share. We stand on the shoulders of giants. There's no need to recreate the wheel if it already exists.
 
-## Familiarize with published plugins
-
-Let's familiarize ourselves with jQuery plugins by investigating a couple.  How do we use them?
-
-### Research existing plugins (T/P/S)(10/20)
-- [isotope.metafizzy.co](http://isotope.metafizzy.co)
-- [packery.metafizzy.co](http://packery.metafizzy.co)
-- [tablesorter](http://tablesorter.com/docs/#Introduction)
-
-Look at the docs in these plugins. Think about - What does it take to utilize a plugin?
-
-> Steps for utilizing
-- include/install
-- initialize/configure/customize
-
-
 ## Basic anatomy of a jQuery Plugin?
 
-What else can we expect from jQuery plugins?  How do we use them?
+What can we expect from jQuery plugins?  How do we use them?
 
-### Group work: (10/10: 20/40)
+### You do: Research plugins (20/30)
 
 #### Individually for 10m:
 
 Start in https://learn.jquery.com/plugins/
 
-Review some random plugins:
+Review the below plugins:
+- [isotope.metafizzy.co](http://isotope.metafizzy.co)
+- [packery.metafizzy.co](http://packery.metafizzy.co)
+- [tablesorter](http://tablesorter.com/docs/#Introduction)
+
+Think and partner up to answer:
 - look for commonality
 - look for patterns
 - look for convention
+- look for installation/configuration
 
 #### Break into groups for 10m.  
 
```

Comment:
Formatting's a little off for the questions. Might just need some additional spaces after each line.